CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
+		name = "Withdraw from San Marino"
+		prestige = -25
+		random_country = {
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -50 }
+        	}
+        743 = { add_province_modifier = { name = generic_uprising duration = 365 } }
+		ai_chance = { factor = 0.8 }
+	}
+
+}
+
+#PCI-PSI Break
+country_event = {
+	id = 8020314
+	title = "PCI-PSI Break"
+	desc = EVTDESC8020314
+	picture = "togliatti_nenni"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1957
+		NOT = { year = 1992 }
+		
+		government = democracy
+		is_vassal = no
+		war = no
+		
+		exists = RUS
+		RUS = { government = proletarian_dictatorship }
+		
+		HUN = { OR = { government = proletarian_dictatorship1 government = proletarian_dictatorship } }
+		
+		has_global_flag = hungarian_revolution_1956
+		OR = { has_country_flag = 1953_ITA_Legge_Truffa has_country_flag = 1953_ITA_Centrism }
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Interesting..."
+		
+		any_pop = {
+			scaled_consciousness = { factor = 2 ideology = communist }
+			scaled_consciousness = { factor = 2 ideology = communist_social }
+			scaled_consciousness = { factor = 3 ideology = socialist }
+		}
+		any_pop = { 
+			limit = { strata = middle }
+			ideology = { factor = 0.05 value = socialist }
+			ideology = { factor = -0.05 value = communist }
+		}
+
+	}
+	
+}
+
+#Italian Economic Miracle
+country_event = {
+	id = 8020315
+	title = "The Italian Economic Miracle"
+	desc = EVTDESC8020315
+	picture = "italy_economic_boom"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1955
+		NOT = { year = 1992 }
+		
+		government = democracy
+		is_vassal = no
+		war = no
+		
+		has_country_modifier = democratic_bloc
+		AND = { has_country_flag = ITA_Casmez has_country_flag = ITA_Legge_Stralcio has_country_flag = ITA_Piano_Fanfani }
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Amazing"
+		
+		prestige = 100
+		
+		any_pop = { 
+		        limit = { OR = { strata = rich strata = middle } }
+				consciousness = -4
+				militancy = -4
+		}		
+		
+        set_country_flag = italy_dirigisme
+        add_country_modifier = { name = dirigisme duration = 7300 }
+		add_country_modifier = { name = economic_boom duration = 7300 }
+	}
+	
+}
+
+#1958 elections
+country_event = {
+	id = 8020316
+	title = "$YEAR$ Italian General Election"
+	desc = EVTDESC8020316
+	picture = "ITA_1948_elections"
+	
+	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1958
+		NOT = { year = 1963 }
+		
+		government = democracy
+		war = no
+		is_vassal = no
+		
+		OR = { has_country_flag = 1953_ITA_Centrism has_country_flag = 1953_ITA_Legge_Truffa has_country_flag = 1953_ITA_FDP }
+	}
+
+	mean_time_to_happen = { months = 2 }
+
+	option = {
+		name = "Centre-Left Coalition (DC-PSDI)"
+		ruling_party_ideology = big_tent
+		any_pop = { 
+			ideology = { factor = -0.30 value = conservative }
```

Comment:
Please do not remove ideology support as that causes crashes.

Reply:
How do I make it so an ideology looses support then?

Reply:
Increasing the ideology of one would reduce the ideology of the others. So conservative ideology is going to be reduced even without this.